### PR TITLE
Added Base Field search

### PR DESCRIPTION
**Pull request by limarta**
Added feature to search by moduli fields for Belyi maps.

### CODE REVIEW

File: lmfdb/belyi/main.py [hunk 1]
```diff
@@ -764,8 +766,14 @@ def __init__(self):
             label="Geometry type",
             knowl="belyi.geometry_type",
             options=[("", "")] + list(geometry_types_dict.items()))
+        field = TextBox(
+            name="field",
+            label="Moduli field",
+            knowl="ag.base_field",
```
Comment (edgarcosta) [inline lmfdb/belyi/main.py:772]: This points to the wrong knowl

Comment (edgarcosta) [reply lmfdb/belyi/main.py:772]: The knowl should be: belyi.moduli_field

**Comment by edgarcosta** [comment]
@jvoight and @SamSchiavone  are figuring out what is the right knowl to use here and in the column headings.
Given that belyi is still restricted to beta I would propose to go ahead and merge this.

**Comment by SamSchiavone** [comment]
Yeah, there's some field of definition vs. field of moduli and pointed vs. unpointed variants of these that needs to get sorted out. For the moment, maybe it's best to just change the knowl to `belyi.base_field`, which already exists.

**Comment by edgarcosta** [comment]
Superseded by #4534

**Pull request closed without merging**